exercises/isbn-verifier/isbn_verifier_test.py

```diff
@@ -0,0 +1,51 @@
+import unittest
+
+from isbn_verifier import verify
+
+
+# Tests adapted from `problem-specifications//canonical-data.json` @ v2.0.0
```

Comment:
Apologies for the delay in reviewing, but since this was written the canonical data has been updated. Can you please update the tests to v2.2.0?